plone/app/tiles/modifiers.py

```diff
@@ -67,6 +67,8 @@ def manage_addCleanTileAnnotations(self, id, title=None, REQUEST=None):
         REQUEST['RESPONSE'].redirect(self.absolute_url() + '/manage_main')
 
 
+ANNOTATION_KEY_PREFIXES = [ANNOTATIONS_KEY_PREFIX,  # plone.tiles default
+                           'plone.tiles.scale']  # collective.cover scales
```

Comment:
@rodfersou In theory that annotation key could be "componentized", but that would not be worth of effort. And because collective.cover already uses "plone.tiles.scale.[uuid]" as key, there is no danger of conflicts.